addons/customGear/launchers/CfgWeapons.hpp

```diff
@@ -0,0 +1,102 @@
+class CfgWeapons {
+    // Base Classes
+    class Launcher;
+    class Launcher_Base_F: Launcher {
+        class WeaponSlotsInfo;
+    };
+    
+    // CUP Inheritances
+    class CUP_launch_M136_Loaded: Launcher_Base_F {
+        class WeaponSlotsInfo;
+        class EventHandlers;
+    };
+    class CUP_launch_M136: CUP_launch_M136_Loaded {};
+    class CUP_launch_M136_Used: CUP_launch_M136_Loaded {};
+    class CUP_launch_M72A6_Loaded: Launcher_Base_F {
+        class WeaponSlotsInfo;
+        class EventHandlers;
+    };
+    class CUP_launch_M72A6: CUP_launch_M72A6_Loaded {};
+    class CUP_launch_M72A6_Used: CUP_launch_M72A6_Loaded {};
+    
+    // M136 AT4 Confined Space-Reduced Sensitivity
+    class potato_launch_M136A1_CS_Loaded: CUP_launch_M136_Loaded {
+        author = "Potato";
+        scope = 1;
+        scopeArsenal = 1;
+        displayName = "M136A1 CS-RS";
+        baseWeapon = "potato_launch_M136A1_CS_Loaded";
+        jsrs_soundeffect = "JSRS2_Distance_Effects_Launcher";
+        AGM_UsedTube = "AGM_launch_M136_Used_F";
```

Comment:
```suggestion
```
It was a great mod, but we probably don't need configs for it 😜

Reply:
LOL, fair point